clients/admin-ui/src/features/common/AccordionTree.tsx

```diff
@@ -0,0 +1,90 @@
+import {
+  Accordion,
+  AccordionButton,
+  AccordionIcon,
+  AccordionItem,
+  AccordionPanel,
+  Box,
+  Button,
+  ButtonGroup,
+  Text,
+} from "@fidesui/react";
+import { Fragment, useState } from "react";
+
+import { TreeNode } from "./types";
+
+const ActionButtons = () => (
+  <ButtonGroup size="xs" isAttached variant="outline">
+    <Button>Edit</Button>
+    <Button>Delete</Button>
+  </ButtonGroup>
+);
+
+interface Props {
+  nodes: TreeNode[];
+}
+const AccordionTree = ({ nodes }: Props) => {
+  const [hoverNode, setHoverNode] = useState<TreeNode | undefined>(undefined);
```

Comment:
@ssangervasi I'm having some hover troubles, wondering if you could help me out? Things are a little funky since this is a button group inside of something else that is clickable. so when I originally put the `onMouseOut` with the `onMouseEnter` there'd be some weird behavior. I tried moving the `onMouseOut` around but the buttons still disappear, sometimes just when you go to click them which is no good :) 

I removed the `onMouseOut` for now, but that leads to this behavior where if you leave the component there's a leftover button hanging out, when I think we want it gone.

https://user-images.githubusercontent.com/24641006/179300780-818aed84-b85e-4ff4-839b-ba162377725a.mov

Reply:
> Things are a little funky since this is a button group inside of something else that is clickable

Looks like you've updated some of this since this comment, keeping the accordion button to the left. That seems the right approach, though now it could use some padding between the accordion button and the action buttons.

I'm still experimenting with the hover out.

Reply:
Oh yeah I remember this problem! Mouse In/Out bubble, Mouse Enter/Leave [do not](https://developer.mozilla.org/en-US/docs/Web/API/Element/mouseleave_event). So In/Out are useful at the root of some dom where you want a single place to listen for mouse movement among the children, while Enter/Leave are useful when asking if the mouse is moving on _this_ element. So using `onMouseLeave` does what you want.

That being said, if we were getting into performance concerns (which this is not) using in/out at the root of the tree is an optimization to reduce how many callbacks are firing as the user mouses around.

Reply:
oh wow! TIL—thanks for such a nice solution!